OTRGui/src/impl/extractor/extractor.cpp

```diff
@@ -50,10 +50,11 @@ void BuildOTR(const std::string output) {
 	setCurrentStep("Done!");
 
 	if (output == ".") return;
```

Comment:
I tested this on Windows and it looks like OTRGUI ends up here with an `output` that is `.` so this function ends. The .otr file actually appears to be generated by OTRExporter instead in `Main.cpp` `ExporterProgramEnd()` where it is hardcoded as `oot.otr`, and this copies the file elsewhere if needed, but I guess since the otr file is generated at the root next to OTRGUI, this skips needing to copy the file.

Reply:
Thanks for testing this! I pushed up a change that I would think should handle this case, but it doesn't appear to be working